pkg/ecctl/init.go

```diff
@@ -55,6 +55,35 @@ const (
 	esspInfraChoice
 )
 
+const (
+	_ = iota
+	gcpUsCentral1Choice
```

Comment:
This is not realistic. With the pace regions are added we will need to update it very frequently . TBH It's already outdated :) there's a new region `us-east-2` coming really shortly
Additionally the first ESSP multi-region is coming so we need to keep that in mind as well.